### PR TITLE
Remove direct SSH/SCP and use machine.communicate

### PR DESCRIPTION
**Pull request by bexelbie**
Modify how keys are downloaded

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
LGTM. However I have not tested this yet.

File: lib/vagrant-service-manager/command.rb [hunk 1]
```diff
@@ -192,14 +175,14 @@ def execute_docker_info
...
-            hprivate_key_path = machine.ssh_info[:private_key_path][0]
+            # Create the directory
+            Dir.mkdir(secrets_path)
 
             # copy the required client side certs from inside the box to host machine
             @env.ui.info("# Copying TLS certificates to #{secrets_path}")
-            self.copy_from_box(hIP, hport, husername, hprivate_key_path, "/home/vagrant/.docker/ca.pem", "#{secrets_path}")
-            self.copy_from_box(hIP, hport, husername, hprivate_key_path, "/home/vagrant/.docker/cert.pem", "#{secrets_path}")
-            self.copy_from_box(hIP, hport, husername, hprivate_key_path, "/home/vagrant/.docker/key.pem", "#{secrets_path}")
+            machine.communicate.download("/home/vagrant/.docker/ca.pem", "#{secrets_path}")
```
Comment (hferentschik) [inline lib/vagrant-service-manager/command.rb:183]: +1, using the Vargrant ways :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 - Fixed missing word for plugin installation in README @budhrg
 - Fix links, typos, formatting in CONTRIBUTING.md @budhrg
 - Fix #16 and #72: Enable private networking for VirtualBox if not set @budhrg
+- Remove manually scp for TLS keys and use machine.communicate.download @bexelbie
```
Comment (navidshaikh) [inline CHANGELOG.md:8]: We should put the new changelogs at the top. Fine for now, keep a note henceforth.

File: lib/vagrant-service-manager/command.rb [hunk 3]
```diff
@@ -192,14 +175,14 @@ def execute_docker_info
           # First, get the TLS Certificates, if needed
           if !File.directory?(secrets_path) then
 
-            # Get the private key
-            hprivate_key_path = machine.ssh_info[:private_key_path][0]
+            # Create the directory
+            Dir.mkdir(secrets_path)
```
Comment (navidshaikh) [inline lib/vagrant-service-manager/command.rb:179]: Do we need to explicitly create a new directory? Did you check if `machine.communicate.download` creates one if not already exist ?

Comment (navidshaikh) [inline lib/vagrant-service-manager/command.rb:179]: Okay - I tested this, we need to create the directory explicitly.

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**